cves/kernel/CVE-2020-36558.yml

```diff
@@ -135,10 +126,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
+  code: false
   code_answer:
-  fix:
-  fix_answer:
+  fix: true
+  fix_answer: The bug was discovered by a coverage based system call fuzzer.
```

Comment:
Similar to my comments on the other CVE, add a code answer, and I don't think fix is true here.

Reply:
Addressed